src/services/ApiConfigStore.ts

```diff
@@ -0,0 +1,28 @@
+// our own mini redux store for storing api configs :))
+
+import { ConfigurationParameters, Configuration } from './api';
+
+class ApiConfigStoreClass {
+  private configParams: ConfigurationParameters = {
+    // TODO: load this from a config instead
+    basePath:
+      process.env.NODE_ENV === 'development'
+        ? 'http://localhost:3000'
```

Comment:
Change to 3001 instead

Reply:
do you mind just pushing a fix since you have something working already?

Reply:
Sure. I'm thinking about integrating these new functions in another PR since it feels a little out of scope in this PR. Is that okay?

Reply:
Or maybe it's better to have an example of it being used

Reply:
i think either way's fine - we could also just merge this in for now.

Reply:
I'll make sure the calendar still works after this change at least. Just to show definitively that it works

Reply:
ok sounds good to me